doc/README.md

```diff
@@ -89,6 +89,16 @@ Alternatively, you can use some specific targets:
 Also note the `-with-doc yes` option of `./configure` to enable the
 build of the documentation as part of the default make target.
 
+If you're editing Sphinx documentation, set SPHINXWARNERROR to 0
+to avoid treating Sphinx warnings as errors.  Otherwise, Sphinx quits
+upon detecting the first warning.  You can set this on the Sphinx `make`
+command line or as an enviroment variable:
+
+- `make sphinx SPINXWARNERROR=0`
+
+- `export SPHINXWARNERROR=0`
+    :
+  `make sphinx`
```

Comment:
If you want to put several lines of code, you can just use code blocks (three backquotes to start it and to end it).

Reply:
tildes also work, eg
```markdown
not code
~~~ocaml
let x = 2 (* code *)
~~~
```